src/common/config.h

```diff
@@ -9,13 +10,29 @@ namespace raftcpp {
 
 class Config final {
     public:
-    static Config From(const std::string &config_str) { return Config(); }
+    static Config From(const std::string &config_str) {
+        static Config config;
+        config.endpoints.clear();
+        std::regex reg(R"((\d{1,3}(\.\d{1,3}){3}:\d+))");
+        std::sregex_iterator pos(config_str.begin(), config_str.end(), reg);
+        decltype(pos) end;
+        for (; pos != end; ++pos) {
+            config.push_back(Endpoint(pos->str()));
+        }
+        return config;
+    }
 
     Endpoint GetSelfEndpoint() const {}
 
-    std::vector<Endpoint> GetAllEndpoints() const {}
+    void push_back(const Endpoint &endpoint) {
```

Comment:
1. `push_back` should be a private method.
2. It should be renamed to `PushBack()`.

Reply:
Done